scientific-computing-with-python/learn-data-structures-by-building-the-shortest-path-algorithm/shortest-path-algorithm.py

```diff
@@ -0,0 +1,51 @@
+my_graph = {
+    'A': [('B', 1), ('C', 6)],
+    'B': [('A', 1), ('D', 3), ('E', 1), ('F', 5)],
+    'C': [('A', 6), ('D', 4)],
+    'D': [('B', 3), ('C', 4), ('E', 2), ('G', 3)],
+    'E': [('B', 1), ('D', 2), ('F', 9)],
+    'F': [('B', 5), ('E', 9), ('G', 8)],
+    'G': [('D', 3), ('F', 8)]
+}
+
+def shortest_path(graph, start, end = ''):
+    unvisited = []
+    distances = {}
+    paths = {}
+    inf = float('inf')
+    for node in graph:
+        unvisited.append(node)
+        paths[node] = []
+        if node == start:
+            distances[node] = 0
+        else:
+            distances[node] = inf
+    
+    current = start
+    paths[current].append(current)
+    
+    while True:
```

Comment:
```suggestion
    while unvisited:
        current = min(unvisited, key=distances.get)
```
To conform with other changes.